tsconfig.json

```diff
@@ -1,9 +0,0 @@
-{
```

Comment:
I think this was needed for the `build:dev` command - I'm not sure if it's widely used by the team, it was something I added when doing the backend-core conversion to TS that allows for running ts build in watch mode across all backend dependencies - possibly not applicable anymore with nx.

Reply:
Nice!